src/authorization/AclManager.ts

```diff
@@ -2,6 +2,9 @@ import type { ResourceIdentifier } from '../ldp/representation/ResourceIdentifie
 
 /**
  * Handles where acl resources are stored.
+ *
+ * Solid, §4.3.3: "A given Solid resource MUST NOT be directly associated with more than one ACL auxiliary resource.
```

Comment:
I suggest adding a URL to these? Or do we have global refs somewhere? (but even then)

Reply:
So for every spec reference, add an URL linking to the most specific part possible? (And then upgrade when there are more specific identifiers as mentioned by Sarven below).